src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -3,30 +3,76 @@
 import core.basesyntax.dao.UserDao;
 import core.basesyntax.model.User;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
 
 public class UserDaoImpl extends AbstractDao implements UserDao {
     public UserDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public User create(User entity) {
-        return null;
+    public User create(User user) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(user);
```

Comment:
- to make code consistent use session.save() + hibernate cascade instead of jpa

Reply:
Done